tools/rosbag_storage/include/rosbag/message_instance.h

```diff
@@ -93,7 +93,6 @@ class ROSBAG_STORAGE_DECL MessageInstance
     //! Size of serialized message
     uint32_t size() const;
 
-private:
```

Comment:
I am a bit concerns of publically exposing these private members. Can you please elaborate why you made that change and what was missing from the public interface to not require this.